src/test/resources/Logout.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Logout
+  An a user
+  I want to logout system
+  So I can use function logout
```

Comment:
So I can use function logout, c chỉ cần viết là` So I can logout the app` là được rồi